Automattic/wp-calypso
# Plans: Remove `plansFeatureList` test

This removes the test from #3316 and reverts to showing the description for each plan.

**Testing**

* Visit http://calypso.localhost:3000/plans
* Select a site to view the plans
* Run localStorage.setItem( 'ABTests', '{"plansFeatureList_20040202":"list"}' ); in your console
* Refresh and confirm you see the description text under the plans
